lib/customer.rb

```diff
@@ -2,9 +2,11 @@
 class Customer
 
 attr_reader :customer_dishes
+attr_reader :order_msg
```

Comment:
these two could be on the same line if a comma was used. this would avoid repeating attr_reader